espnet2/fileio/read_text.py

```diff
@@ -32,7 +32,7 @@ def read_2columns_text(path: Union[Path, str]) -> Dict[str, str]:
                 k, v = sps
 
             if k in data:
-                raise RuntimeError(f"{k} is duplicated ({path}:{linenum})")
+                logging.warning(f"{k} is duplicated ({path}:{linenum})")
```

Comment:
I am not confident about this change. @sw005320 Can you double-check this?

Reply:
We could keep it as an error, although the warning is useful in large multi-corpora work if you accidentally have some overlap and filtering manually is too expensive.